app/messaging/inbound/process-ingestion.js

```diff
@@ -0,0 +1,24 @@
+const util = require('util')
+const { validateIngestionMessage } = require('./ingestion-schema')
+const { ingestDocument } = require('../../lib/knowledge-ingestion')
+
+const processIngestion = async (message, receiver) => {
+  try {
+    const { body } = validateIngestionMessage(message.body)
+    console.log(`Processing ingestion: ${util.inspect(body)}`)
+
+    await ingestDocument(body.document_id)
+
+    console.log(`Ingestion of document ${body.document_id} complete`)
```

Comment:
is this console.log needed here?

Reply:
Yes it's needed here for logging when we're in deployed on ADP.